_posts/2021-12-20-changelog-162.markdown

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title: "IntelliJ Rust Changelog #162"
+date: 2021-12-20 13:00:00 +0300
+---
+
+
+## New Features
+
+* [#8145] Take into account [const generics defaults] in const evaluation
+
+* [#8070] Suggest private items when completing code in the debugger
+
+* [#8192] Parse and annotate [`~const` syntax]
+
+## Fixes
+
+* Procedural macro expansion fixes:
+
+  * [#8271] Fix expansion of `sqlx` procedural macros with `offline` feature
+
+  * [#8233] fix expansion of proc macros that weirdly mix token spans (e.g. `yew::html!`)
+
+  * [#8203] fix expansion of proc macros with a float literal in a body
+
+  * [#7947] Fix exception when expanding macros in parallel and enable it by default
```

Comment:
Maybe move it into `Performance Improvements` section? Also, I'd add that there was an attempt to make procedural macro expansion parallel but it was reverted because of the bug